### PR TITLE
Rule Config for Kinesis Data Firehose IPs

### PR DESCRIPTION
**Pull request by ocrawford555**
This PR adds an example `RuleConfig` to make CFRipper not flag valid AWS IPs when scanning a template containing a security group (usually for AWS Redshift). More information about access between Kinesis Data Firehose and Redshift can be found on this link:

https://docs.aws.amazon.com/firehose/latest/dev/controlling-access.html#using-iam-rs

Changes:

- updated README to include a simple HOWTO on using the `--rules-config-file` flag on the CLI
- Firehose specific rule filter for the `EC2SecurityGroupOpenToWorldRule` added, and shown in a complete example
- unit tests added

There should be **no** functional changes to CFRipper from this PR. The filter is not automatically applied unless added to the `RuleConfig` when running CFRipper.

Additionally, the CLI is not currently working as expected when testing locally. This is to be raised as a separate issue and fixed in another PR (TL;DR --> we are passing IO objects, but the functions are expecting strings).

### CODE REVIEW

**Review by ignaciobolonio — APPROVED** [review]
👍

**Review by oscarbc96 — APPROVED** [review]
Very nice!! 👏

File: cfripper/config/rule_configs/firehose_ips.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+config = Config(
+    ...
+    rules_config=RULES_CONFIG,
+)
+```
+"""
+
+# Adapted from https://docs.aws.amazon.com/firehose/latest/dev/controlling-access.html
+FIREHOSE_IPS = [
```
Comment (jsoucheiron) [inline cfripper/config/rule_configs/firehose_ips.py:22]: Could you use a set, sort them and add a comment on each one with the region name?

Comment (ocrawford555) [reply cfripper/config/rule_configs/firehose_ips.py]: Done 👍

**Review by ignaciobolonio — APPROVED** [review]

**Review by jsoucheiron — APPROVED** [review]

**Pull request merged**